src/DDTrace/Tracer.php

```diff
@@ -191,7 +192,7 @@ public function startSpan($operationName, $options = [])
         }
 
         // This dynamic property ensures that the scope manager does not recognize startSpan() spans without activation
-        $internalSpan->ddtrace_scope_activated = false;
+        ObjectKVStore::put($internalSpan, 'ddtrace_scope_activated', false);
```

Comment:
I don't know what's going on here, but the comment no longer matches at the least.

Reply:
ObjectKVStore is just a wrapper around weakmap on PHP 7.4+ and dynamic properties before, so it sort of still applies. Not sure what to call it now?